### PR TITLE
Rubocop layout fixes

### PR DESCRIPTION
**Pull request by juanszymczak**
remove extra spacing
remove trailing whitespace
use _ or _options as an argument name to indicate that it won't be used.
clean extra empty line detected at class body beginning.
Align the elements of an array literal if they span more than one line.
remove trailing blank lines detected.
add missing space between {, |, and }.
Align the parameters of a method call if they span more than one line.
add space missing after comma.
align if indentation.

### CODE REVIEW

File: lib/bitex/payment.rb [hunk 1]
```diff
@@ -1,22 +1,24 @@
 module Bitex
   class Payment
-    attr_accessor :id, :user_id, :amount, :currency_id, :expected_quantity,
-      :previous_expected_quantity, :confirmed_quantity, :unconfirmed_quantity,
-      :valid_until, :quote_valid_until, :last_quoted_on, :status, :address,
-      :settlement_currency_id, :settlement_amount, :keep, :merchant_reference,
-      :customer_reference
+    attr_accessor %i[
```
Comment (tmsrjs) [inline lib/bitex/payment.rb]: TypeError@circleci

**Comment by tmsrjs** [comment]
Cool. Please rebase onto or merge master to resolve conflicts.

**Comment by juanszymczak** [comment]
Rebased

**Comment by juanszymczak** [comment]
rebased

File: lib/bitex/payment.rb [hunk 2]
```diff
@@ -3,23 +3,24 @@ module Bitex
...
-            raw_value
-          end
+          value =
+            if [:valid_until, :quote_valid_until, :last_quoted_on].include?(key.to_sym)
+              Time.at(raw_value)
+            else
+              raw_value
+            end
+
+          thing.send("#{key}=", value) rescue nil
```
Comment (tmsrjs) [inline lib/bitex/payment.rb]: Please do a more thorough review of the diff before committing. This is undoing some of your own changes from PR #8.

File: lib/bitex/payment.rb [hunk 3]
```diff
@@ -3,23 +3,22 @@ module Bitex
...
+                  :settlement_currency_id, :settlement_amount, :keep, :merchant_reference, :customer_reference
 
     # @visibility private
     def self.from_json(json)
       new.tap do |thing|
-        json.each do |key,raw_value|
+        json.each do |key, raw_value|
           next if raw_value.nil?
 
-          value = if %i[valid_until quote_valid_until last_quoted_on].include?(key.to_sym)
```
Comment (tmsrjs) [inline lib/bitex/payment.rb]: Again, undoing your own changes.

**Pull request closed without merging**